src/main/java/picard/util/LiftOverIntervalList.java

```diff
@@ -161,6 +164,19 @@ protected int doWork() {
         }
 
         toIntervals.sorted().write(OUTPUT);
-        return anyFailed ? 1 : 0;
+
+        if (REJECT != null) {
+            rejects.write(REJECT);
+        }
+
+        LOG.info(String.format("Liftover Complete. \n" +
+                        "%d of %d intervals failed (%g%%) to liftover, encompassing %d of %d bases (%g%%).",
+                rejects.getIntervals().size(), intervalList.getIntervals().size(),
+                100 * rejects.getIntervals().size() / (double) intervalList.getIntervals().size(),
+                rejects.getBaseCount(), intervalList.getBaseCount(),
+                100 * rejects.getBaseCount() / (double) intervalList.getBaseCount()
```

Comment:
Pull `rejects.getBaseCount()` and `intervalList.getBaseCount()` into local vars to avoid the repeat computation? It ends up [looping over](https://github.com/broadinstitute/htsjdk/blob/master/src/main/java/htsjdk/samtools/util/Interval.java#L130) all the intervals on each call.